### PR TITLE
Document type aliases

### LINKED ISSUE #26: Add rustdoc to type aliases
See https://github.com/ewasm/ewasm-rust-api/pull/25#discussion_r239223392

### PR DESCRIPTION
**Pull request by jakelang**
closes #26

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -132,12 +132,25 @@ pub struct Bytes32 {
...
+/// Alias representing a storage value.
 pub type StorageValue = Bytes32;
+
+/// Alias representing a log topic.
 pub type Topic = Bytes32;
+
+/// Alias representing a keccak or SHA-256 hash.
 pub type Hash = Bytes32;
+
+/// Alias representing a block's difficulty.
```
Comment (axic) [inline src/lib.rs]: Do we need alias in the text?

Comment (jakelang) [reply src/lib.rs]: Would you rather it be `Type`?

Comment (axic) [reply src/lib.rs]: I'm calling it a type is fair. That's how the user uses it.

File: src/lib.rs [hunk 2]
```diff
@@ -132,12 +132,25 @@ pub struct Bytes32 {
...
+/// Alias representing a storage key.
 pub type StorageKey = Bytes32;
+
+/// Alias representing a storage value.
 pub type StorageValue = Bytes32;
+
+/// Alias representing a log topic.
 pub type Topic = Bytes32;
+
+/// Alias representing a keccak or SHA-256 hash.
```
Comment (axic) [inline src/lib.rs]: Please style this as "Keccak-256 or SHA-256 hash".

**Pull request merged**